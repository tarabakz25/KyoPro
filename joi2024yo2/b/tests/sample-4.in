10 5 4
2 1
2 5
2 4
2 3
2 4
2 2
2 2
2 4
2 2
2 1
3 2 7
1 1 7
2 1 10
5 5 8
