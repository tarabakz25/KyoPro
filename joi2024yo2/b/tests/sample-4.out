11
13
17
8
